Recipes/TrainingDemo.Workflows.Sample.recipe.json

```diff
@@ -19,52 +19,93 @@
       "name": "WorkflowType",
       "data": [
         {
-          "WorkflowTypeId": "4vtt0g09pbnt8x72gnc6557smk",
-          "Name": "Person Page Content Item Created Logger",
+          "WorkflowTypeId": "4nrcf82q9wzwt2kf7p7dz2245b",
+          "Name": "Manage Persons Permission Checker",
           "IsEnabled": true,
           "IsSingleton": false,
           "LockTimeout": 0,
           "LockExpiration": 0,
           "DeleteFinishedWorkflows": false,
           "Activities": [
             {
-              "ActivityId": "4jgzsrtras0wt5byehn5s9j04h",
-              "Name": "LogTask",
-              "X": 290,
+              "ActivityId": "4e4nwk4csmmgrr3gstrn726s4f",
+              "Name": "ManagePersonsPermissionCheckerTask",
+              "X": 330,
               "Y": 0,
               "IsStart": false,
               "Properties": {
                 "ActivityMetadata": {
                   "Title": null
                 },
-                "LogLevel": 4,
-                "Text": {
-                  "Expression": "{{ \"Expected non-error -\" | t }} {{ User.Identity.Name }} {{ \"created a new Person Page content item.\" | t }} "
+                "UserName": {
+                  "Expression": "{{ User.Identity.Name }}"
                 }
               }
             },
             {
-              "ActivityId": "4tr7a58h1qz7jvg2vz37hgprn1",
-              "Name": "ContentCreatedEvent",
+              "ActivityId": "4972p06gpmje4ycny23z6aqt41",
+              "Name": "ContentPublishedEvent",
```

Comment:
Let's do this during login instead, it's more applicable. With a content even it's a contrived example.

Reply:
Using the User LoggedIn event was my original idea too, however it doesn't work that way because it has some kind of bug and is not able to detect that the user is actually logged in. 

`fail: OrchardCore.Workflows.Services.WorkflowManager[0]
      An unhandled error occurred while executing an activity. Workflow ID: '7'. Activity: '4e4nwk4csmmgrr3gstrn726s4f', 'ManagePersonsPermissionCheckerTask'. Putting the workflow in the faulted state.
      System.ArgumentNullException: Value cannot be null. (Parameter 'userName')
         at Microsoft.AspNetCore.Identity.UserManager`1.FindByNameAsync(String userName)...`

If I recall correctly it was even mentioned in one of the training videos as well. So I would rather keep the ContentPublished event if that is okay for you.